pom.xml

```diff
@@ -69,39 +69,4 @@
         <connection>scm:git:git@github.com:zalando/logbook.git</connection>
         <developerConnection>scm:git:git@github.com:zalando/logbook.git</developerConnection>
     </scm>
-    <profiles>
```

Comment:
I removed this but maybe the jars should be made available publicly. Was initially planning to deploy to our own nexus

Reply:
Just a small question - for us that would be: https://nexus.tools.ops.smartbear.io/nexus/#welcome ?
I could find information about few nexus instances - but some seem to be just for particular projects (for example just for TM4J).

Reply:
Yeah thats the URL, I was thinking of using this https://github.com/marketplace/actions/generate-settings-xml-for-maven-builds for setting up a settings file in github actions for when we want to deploy to our own repository.